src/presentation/test/integration/post-transition/landing-page.test.ts

```diff
@@ -0,0 +1,36 @@
+import request from "supertest";
```

Comment:
just noticed we have different ways of spelling the folders for post transition
src/presentation/controller/postTransition/url.ts
src/presentation/test/integration/post-transition/landing-page.test.ts

the test one seems to be out of sync with the others